documentation/API-design-guidelines.md

```diff
@@ -1322,29 +1324,52 @@ An instance-based subscription is a subscription indirectly created, additionall
 
 Providing this capability is optional for any CAMARA API depending on UC requirements.
 
-If this capability is present in CAMARA API, `webhook` object attribute **must** be used in the POST request. The `webhook` object contains following attributes :
+If this capability is present in CAMARA API, following attributes **must** be used in the POST request :
 
 | attribute name | type | attribute description | cardinality |
 | ----- |	-----  |	 -----  | -----  | 
-| notificationUrl | string | https callback address where the notification must be POST-ed | mandatory |
-| notificationAuthToken | string | OAuth2 token to be used by the callback API endpoint. It MUST be indicated within HTTP Authorization header e.g. Authorization: Bearer $notificationAuthToken | optional |
+| sink | string | https callback address where the notification must be POST-ed | mandatory |
+| sinkCredential | object | Sink credential provides authentication or authorization information necessary to enable delivery of events to a target. In order to be updated in future this object is polymorphic | optional |
 
-##### Instance-based (implicit) subscription example
+Several types of `sinkCredential` could be available in future but for now only access token credential are managed.
 
-```javascript
+``sinkCredential`` attributes table (must be access token for now):
+
+| attribute name | type | attribute description | cardinality |
+| ----- |	-----  |	 -----  | -----  | 
+| credentialtype | string | Type of the credential - MUST be set to `ACCESSTOKEN` for now | mandatory |
+| accessToken | string | Access Token granting access to the POST operation to create notification | mandatory |
+| accessTokenExpireUtc | string date-time | An absolute UTC instant at which the access token shall be considered expired. | mandatory |
+| accessTokenType | string | Type of access token - MUST be set to `Bearer` for now | mandatory |
+
+
+##### Instance-based (implicit) subscription examples
+
+If credentials are not required:
```

Comment:
The case "credentials are not required" should not exist. Why should any client worth its security-salt ever expose a callback endpoint without any security? On the Internet this is definitely a no-no. Camara should not suggest no-security to API consumers.
If you think "DMZ" or "Intranet", there also should never be any endpoint without security.
If you do not trust my expertise, then trust Google's security exports. [Zero-Trust](https://cloud.google.com/learn/what-is-zero-trust).

If an subscriber realy realy wants to be insecure then they can set the access-token to "insecure" and ignore the Authorization header at their endpoint.

I strongly suggest removing everything related to "If credentials are not required".

Reply:
I'm fine to remove this example.

BTW, we have a § on the security [here](https://github.com/camaraproject/Commonalities/blob/d5da262ffa8d806a2b4778a9b6e3f3c5af809ee7/documentation/API-design-guidelines.md#security-considerations) .

Reply:
Example removed + added recommendation line 1404